<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParamControler</class>
 <widget class="QWidget" name="ParamControler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Function</string>
   </property>
   <widget class="QRadioButton" name="Motion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>100</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Motion</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Color">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Color</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Temporal Filter</string>
   </property>
   <widget class="QRadioButton" name="IIR">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>100</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>IIR</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Ideal">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>100</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ideal</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Butterworth">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>100</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Butterworth</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="OK">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="Alpha">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Alpha</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Alpha_input">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LambdaC">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>LambdaC</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LambdaC_input">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Freq_low">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Freq_low</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Freq_low_input">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Freq_high_input">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>210</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Freq_high">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Freq_high</string>
   </property>
  </widget>
  <widget class="QLabel" name="Attenuation">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Attenuation</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Attenuation_input">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Notice">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
